### DISCUSSION TITLE
Mini-évent "Maître des jeux"

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2354] [Nouvelles idées]
L'idée consiste à pouvoir rencontrer en chemin un "Maître des Jeux".
On aurait la possibilité d'accepter ou de refuser un duel basé sur un jeu inspiré de pierre-feuille-ciseaux. 

Il suffirait de faire un choix parmi les réactions proposées =
🗡️ Epée : coupe à travers le bouclier
🛡️ Bouclier : protège contre la magie
✨ Magie : surpasse l'épée

victoire : 1000 💰 
égalité : 200 💰 ou une tape sur l'épaule ^^'
défaite : -500 💰

**Comment by Sousoulman** [comment]
Je suis d'accord sur le principe mais peut être baissé les récompenses et les pertes (genre 400 pour une victoire et 200 pour une défaite) car ça reste un Mini-event
